src/elements/user-instructions.md

```diff
@@ -0,0 +1,561 @@
+# User setup instructions
```

Comment:
Skipping in-depth review here, as it's on my to-do list to revise this file

Reply:
Thank you